src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -198,8 +209,22 @@ export function createFormConfig781a(iterations) {
         ),
         path: `disabilities/ptsd-secondary-incident-unit-assignment-${index}`,
         depends: isAnswering781aQuestions(index),
-        uiSchema: secondaryIncidentUnitAssignment.uiSchema(index),
-        schema: secondaryIncidentUnitAssignment.schema(index),
+        uiSchema: secondaryIncidentDescription.uiSchema(index),
```

Comment:
Did this get swapped with `secondaryIncidentUnitAssignment`?

Reply:
Not sure what happened here. Thanks!